fh-messaging/test/jobs/test_metrics_prune_job.js

```diff
@@ -0,0 +1,44 @@
+var sinon = require('sinon');
+var fhlogger = require('fh-logger');
+var metricsPruneJob = require('../../lib/jobs/metrics_prune_job');
+
+var mockAgenda, config, logger;
+var db = {
+  command: function() {}
+};
+
+exports.setUp = function(setUp, assert) {
```

Comment:
Yes this needs to be exported because of how this function is defined in the job [here](https://github.com/feedhenry/fh-messaging/pull/23/files#diff-1579951aae88efe37e1b7b4ee375109fR9)